src/main/java/reactor/core/publisher/FluxOnBackpressureDrop.java

```diff
@@ -158,28 +154,25 @@ public void onComplete() {
 		}
 
 		@Override
-		public boolean isStarted() {
-			return s != null && !done;
-		}
-
-		@Override
-		public boolean isTerminated() {
-			return done;
-		}
-
-		@Override
-		public Object downstream() {
+		public Subscriber<? super T> actual() {
 			return actual;
 		}
 
 		@Override
-		public long requestedFromDownstream() {
-			return requested;
-		}
-
-		@Override
-		public Object connectedInput() {
-			return onDrop;
+		public Object scan(Attr key) {
+			switch (key) {
+				case PARENT:
+					return s;
+				case REQUESTED_FROM_DOWNSTREAM:
+					return requested;
+				case TERMINATED:
+					return done;
+				case CANCELLED:
+					return s == Operators.cancelledSubscription();
```

Comment:
incorrect, but there's no way to check cancellation for this operator it seems...